content/logs/aws.md

```diff
@@ -0,0 +1,133 @@
+---
+title: AWS Log Collection
+kind: Documentation
+autotocdepth: 2
+hideguides: true
+customnav: lognav
+beta: true
+---
+
+## Overview
+
+This documentation explains how to create a Lambda function in AWS to forward S3 logs to Datadog.
+
+S3 logging is used by well known services such as: ELB, S3, CloudTrail, VPC, CloudFront and CloudWatch. There are then all eligible to this method.
```

Comment:
`there are then all eligible to this method` -> This means that we can use a Lambda function in AWS to forward those logs to Datadog.

Reply:
👍